### PR TITLE
Features/pindel

### PR DESCRIPTION
**Pull request by davidorman**

### CODE REVIEW

**Comment by davidorman** [comment]
flake8 tests passed on my machine, the CI server seems to be running more or at least different tests.

**Comment by adamjstewart** [comment]
> flake8 tests passed on my machine, the CI server seems to be running more or at least different tests.

Could just be running a newer version of Flake8 that is more particular.

File: var/spack/repos/builtin/packages/pindel/package.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#     spack install pindel
+#
+# You can edit this file again by typing:
+#
+#     spack edit pindel
+#
+# See the Spack documentation for more information on packaging.
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pindel/package.py]: You can remove all this FIXME stuff below the license.

File: var/spack/repos/builtin/packages/pindel/package.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    homepage = "http://gmt.genome.wustl.edu/packages/pindel/"
+    url      = "https://github.com/genome/pindel/archive/v0.2.5.tar.gz"
+
+    version('0.2.5b8', 'e6de2ffb38ba1cb89351cdccabe78cde')
+    version('0.2.5b6', 'dc2febb18c203f0ef1ba02b7b882e94b')
+    version('0.2.5b5', '73e964bc19de9ab9e6e8a316353e3184')
+    version('0.2.5b4', 'b4aefd538d9f62578f46440c4bce497e')
+    version('0.2.5b1', 'a8e53e8919aa29093db13fad5ede93a5')
+    version('0.2.5a7', '5fb2bac6108547b5d60c38fc66abdfc4')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pindel/package.py]: Are these alpha and beta releases, or do they have some other strange versioning scheme? We don't normally add alpha/beta releases to packages. If you really want them, you should set the most recent stable to release to `preferred=True`.

File: var/spack/repos/builtin/packages/pindel/package.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    version('0.2.5',   'd4568cbb83ec25ef9f9f6f058b30053e')
+
+    depends_on('htslib@1.7:')
+
+    def edit(self, spec, prefix):
+        makefile2 = join_path(self.build_directory, 'Makefile2')
+        copyfile(join_path(self.build_directory, 'Makefile'), makefile2)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pindel/package.py]: What's with Makefile2? It looks like you are running `make` on both. Might want to add a comment explaining what you are doing here and why it is necessary.

File: var/spack/repos/builtin/packages/pindel/package.py (hunk 2 above)
Comment (davidorman) [reply var/spack/repos/builtin/packages/pindel/package.py]: Ill remove the betas

Comment (davidorman) [reply var/spack/repos/builtin/packages/pindel/package.py]: They dont seem to be betas, the 0.2.5b8 release was 2 years after the 0.2.5 release.

Comment (davidorman) [reply var/spack/repos/builtin/packages/pindel/package.py]: Anything else I need to do?

File: var/spack/repos/builtin/packages/pindel/package.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    version('0.2.5b6', 'dc2febb18c203f0ef1ba02b7b882e94b')
+    version('0.2.5b5', '73e964bc19de9ab9e6e8a316353e3184')
+    version('0.2.5b4', 'b4aefd538d9f62578f46440c4bce497e')
+    version('0.2.5b1', 'a8e53e8919aa29093db13fad5ede93a5')
+    version('0.2.5a7', '5fb2bac6108547b5d60c38fc66abdfc4')
+    version('0.2.5',   'd4568cbb83ec25ef9f9f6f058b30053e')
+
+    depends_on('htslib@1.7:')
+    #
+    # This Makefile2 stuff is due to the orignal installer,
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pindel/package.py]: typo: orignal

**Pull request merged**